src/components/CartProductCard/CartProductCard.jsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import { useCartContext, useWishlistContext } from "../../context/context";
+import "./CartProductCard.css";
+function CartProductCard({ productCard }) {
+  const { removeFromCart, changeCartQuantity } = useCartContext();
+  const { Wishlist, addToWishlist } = useWishlistContext();
+  const [moveLoader, setMoveLoader] = useState(false);
+
+  function isPresentInState(data, state) {
```

Comment:
extract these functions into a utility `utils` folder these can be used at different places